src/main/java/org/testng/internal/Parameters.java

```diff
@@ -200,6 +257,21 @@ private static void checkParameterTypes(String methodName,
     }
   }
 
+  private static String prettyFormat(List<Class> classes) {
```

Comment:
Not able to incorporate this because I didn't change `mapping` value type to `<Class<?>>`